# This message is used to set speed to dynamixel

uint8 id
bool torque_enable
---
bool result
